### DISCUSSION TITLE
支持win7 32位系统吗

### DISCUSSION THREAD

**Discussion by ABYSS0918** [bluenviron/mediamtx#3493] [Questions]
### Question

请问支持win7 32位系统吗？

**Comment by aler9** [comment]
Hello, you can run MediaMTX on 32-bit Windows 7 by compiling it from source:

https://github.com/bluenviron/mediamtx?tab=readme-ov-file#standard

If you want to compile it from another machine:

```
CGO_ENABLED=0 GOOS=windows GOARCH=386 go build .
```

[a comment by an automated account is not shown]
